### PR TITLE
SLE15 Add rsh and talk server remove rules to HIPAA profile

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add rules for rsh and talk server to SLE15 HIPAA profile

#### Rationale:

- add existing rules to profile, adapt and test

### CODE REVIEW

File: linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,rhel7,rhel8,rhel9,rhv4,ubuntu2004,wrlinux1019
+prodtype: ol7,ol8,rhel7,rhel8,rhel9,rhv4,ubuntu2004,wrlinux1019,sle15
```
Comment (vojtapolasek) [inline linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml:3]: Please sort prodtypse alphabetically.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you for the contribution, just a small correction to keep everything in order.

File: linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/services/obsolete/r_services/package_rsh-server_removed/rule.yml]: Thanks done it in a97f867

**Review by vojtapolasek — APPROVED** [review]

**Comment by vojtapolasek** [comment]
Please rebase, then it can be merged.

**Comment by teacup-on-rockingchair** [comment]
> Please rebase, then it can be merged.

Done !

**Pull request merged**